__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -227,5 +286,16 @@ exports[`linting typescript.ts fixture matches snapshot 1`] = `
     "ruleId": "@typescript-eslint/no-shadow",
     "severity": 2,
   },
+  {
+    "column": 49,
+    "endColumn": 54,
+    "endLine": 29,
+    "line": 29,
+    "message": "'value' is defined but never used.",
+    "messageId": "unusedVar",
+    "nodeType": "Identifier",
+    "ruleId": "@typescript-eslint/no-unused-vars",
+    "severity": 2,
+  },
```

Comment:
Added to strict-type-checked in 6.0.0: https://github.com/typescript-eslint/typescript-eslint/blob/v6.0.0/packages/eslint-plugin/src/configs/strict-type-checked.ts#L65